### DISCUSSION TITLE
datagriff/2023/10/29/cloud-dev-platform-template

### DISCUSSION THREAD

**Discussion by giscus** [hungovercoders-blog/datagriff#7] [Announcements]
# datagriff/2023/10/29/cloud-dev-platform-template

Creating a Github template to deploy azure infrastructure with terraform in Azure using gitpod

https://blog.hungovercoders.com/datagriff/2023/10/29/cloud-dev-platform-template.html

**Comment by dataGriff** [comment]
# test 

Alright innit. Not your best.
